twisted/protocols/loopback.py

```diff
@@ -286,7 +286,7 @@ def write(self, data):
             self.logFile.write("loopback writing %s\n" % repr(data))
 
     def writeSequence(self, iovec):
-        self.write(b"".join(iovec))
+        self.write("".join(iovec))
```

Comment:
Instead of taking this change out, can you leave it in,
and extend  twisted/conch/test/test_recvline.py
to additionally call Loopback.writeSequence() ?

That way this line will have test coverage.

Your original change is good, and I would hate to lose this change.